### PR TITLE
Content changes for the start page of the check_travel_during_coronavirus flow

### PR DESCRIPTION
**Pull request by gclssvglx**
This change makes the following changes to the start page of the `check_travel_during_coronavirus flow`:

- Adds a `Travelling to England from outside the Common Travel Area` section

- Moves the content from within CTA section into it's own section and adds additional links to the Isle of Man, Jersey, and Guernsey

| Before | After |
|:-----|:-----|
|Screenshot 2022-02-16 at 11 00 41|Screenshot 2022-02-16 at 11 24 52|
|Screenshot 2022-02-16 at 11 00 56|Screenshot 2022-02-16 at 10 59 59|

Trello
- https://trello.com/c/1WBCdfJP/646-consider-changing-smart-answer-start-page-to-clarify-it-is-only-for-people-living-in-england
- https://trello.com/c/L8RbKwmd/682-add-links-to-channel-isles-and-isle-of-man-on-start-page-mvp

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/flows/check_travel_during_coronavirus_flow/start.erb [hunk 1]
```diff
@@ -44,4 +44,13 @@
   <p>If you’re travelling to England from within the Common Travel Area (England, Scotland, Wales and Northern Ireland, Ireland, the Channel Islands and the Isle of Man) and haven’t been anywhere else within the 10 days before you arrive, there are no entry requirements for coming into England.</p>
 
   <p>If you’ve been to a country or territory outside the Common Travel Area within the 10 days before you arrive in England, you must follow the rules for entering England from that country. You can use this tool to find out those rules.</p>
+
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "Travelling to England from outside the Common Travel Area",
+    heading_level: 2,
```
Comment (leenagupte) [inline app/flows/check_travel_during_coronavirus_flow/start.erb:57]: I think this heading should be smaller? Maybe a level 3?

File: app/flows/check_travel_during_coronavirus_flow/start.erb [hunk 2]
```diff
@@ -44,4 +44,13 @@
...
+    font_size: "m",
+    margin_bottom: 4,
+  } %>
+
+  <p>If you live outside the Common Travel Area (England, Scotland, Wales and Northern Ireland, Ireland, the Channel Islands and the Isle of Man) and are travelling to England, check what you need to do to <a href="https://www.gov.uk/guidance/travel-to-england-from-another-country-during-coronavirus-covid-19">travel to England during coronavirus</a>.</p>
```
Comment (leenagupte) [inline app/flows/check_travel_during_coronavirus_flow/start.erb:62]: Not a blocker: Should be a relative link.
```suggestion
  <p>If you live outside the Common Travel Area (England, Scotland, Wales and Northern Ireland, Ireland, the Channel Islands and the Isle of Man) and are travelling to England, check what you need to do to <a href="/guidance/travel-to-england-from-another-country-during-coronavirus-covid-19">travel to England during coronavirus</a>.</p>
```

**Review by leenagupte — COMMENTED** [review]
I have one question about the heading size.

The content and links look good.

File: app/flows/check_travel_during_coronavirus_flow/start.erb (hunk 1 above)
Comment (gclssvglx) [reply app/flows/check_travel_during_coronavirus_flow/start.erb]: Changed to level 3 - thanks @leenagupte  - I missed that in the Google doc - they looked the same size 👍

File: app/flows/check_travel_during_coronavirus_flow/start.erb (hunk 2 above)
Comment (gclssvglx) [reply app/flows/check_travel_during_coronavirus_flow/start.erb]: Updated link - thanks

**Review by leenagupte — APPROVED** [review]
Looks good!

**Pull request merged**